src/azure-cli/azure/cli/command_modules/acs/managed_cluster_decorator.py

```diff
@@ -3931,8 +3936,40 @@ def _get_uptime_sla(self, enable_validation: bool = False) -> bool:
                 raise MutuallyExclusiveArgumentError(
                     'Cannot specify "--uptime-sla" and "--no-uptime-sla" at the same time.'
                 )
+
+            if uptime_sla and self.get_tier() == "free":
```

Comment:
Better define "free" and "standard" as const values in `_consts.py`

Reply:
Fixed.